octodns/provider/cloudflare.py

```diff
@@ -279,7 +283,23 @@ def _data_for_SRV(self, _type, records):
             })
         return {
             'type': _type,
-            'ttl': records[0]['ttl'],
+            'ttl': self._ttl_data(records[0]['ttl']),
+            'values': values
+        }
+
+    def _data_for_URLFWD(self, _type, records):
+        values = []
+        for r in records:
+            values.append({
+                'path': r['path'],
+                'target': r['url'],
+                'code': r['status_code'],
+                'masking': 2,
+                'query': 0,
+            })
+        return {
+            'type': _type,
+            'ttl': 3600,  # ttl does not exist for this type, forcing a setting
```

Comment:
If someone puts a TTL in the (yaml) config for a `URLFWD` type record that doesn't match this value it'll probably show up as a "constant" change that can't be resolved. There's some other [special cases of changes around TTL down in the `_include_change` method](https://github.com/octodns/octodns/blob/5e2f7d43845d48fbf2e92b7bbaf2bb5b0e529b33/octodns/provider/cloudflare.py#L411) that may need expanding if so.

Long shot, but is there any sort of "notes" field on the URLFWD record where we could stuff the TTL and get it back out of the api? Might save some PITA if so.

Reply:
There is no notes field in the CloudFlare API. Compared to the zone API, it is missing a number of things.

I thought I checked the TTL change occurrence. I apparently, didn't. I add the handling in the referred def.

Reply:
I think I handled this in the prescribed manor via the `_include_change` def. Should I leave the magic numbers in place?